Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1485,11 +1508,18 @@ def png(self, file, width=None,height=None,units=None,draw_white_background = Tr
         writer.SetInputConnection(imgfiltr.GetOutputPort())
         writer.SetFileName(file)
         writer.Write()
+        if manager:
+          self.renWin.AddRenderer(manager.renderer)
 
   def cgm(self,file):
```

Comment:
`cgm` is a function that was already in there; it's one of the graphics output methods, though it's currently busted– see #1065. I only modified it to add support for phasing the UI out before output.